### ISSUE TITLE
GHA: Apache setup failure in the php 5.6 runners

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#5790]
## Description of the bug

This popped up today, never happened before:

```
apache2_invoke: Enable module fcgid
apache2_reload: Your configuration is broken. Not restarting Apache 2
apache2_reload: AH00534: apache2: Configuration error: More than one MPM loaded.
```
Probably something with mpm_event vs. mpm_prefork.

Needs a little investigation.

**Comment by indigoxela** [comment]
Whoops, not sure what actually changed, but mod_php and mpm_prefork seem to be on (all of a sudden).

Not too hard to solve, but it's unclear, when this default changed. PHP 5.6 as provided by the Sury repo (on which shivammathur/setup-php relies) work fine with mpm_event.

**Comment by indigoxela** [comment]
Yes, this seems to do the trick. It's a conditional based on php version in our testing matrix:

```
        if: ${{ matrix.php-versions == '5.6' }}
```
Then turn off mod_php and mpm_prefork and turn on mpm_event - the rest is identical to other php versions.

**Comment by indigoxela** [comment]
Neither the action (shivammathur/setup-php) nor the runner image (2.296.2) changed recently, the only explanation is that it has something to do with the new php 5.6 build (5.6.40-62+ubuntu20.04.1+deb.sury.org+1), which has been updated on 2022-09-18.

**Comment by herbdool** [comment]
I think this looks good. All the tests pass whereas on an another PR without this fix, all the 5.6 tests fail (as described above). Getting this fix in is critical for any other PR to fully pass, so I've set the milestone. Pinging @quicksketch to see if you'll be able to merge.

**Comment by quicksketch** [comment]
Thanks @indigoxela! I merged https://github.com/backdrop/backdrop/pull/4209 into 1.x and 1.23.x.
